src/index.ts

```diff
@@ -27,6 +27,7 @@ export {
   TransactionBuilder,
   SponsoringBuilder,
 } from "./walletSdk/Horizon";
+export { KeyManager } from "./walletSdk/KeyManager";
```

Comment:
I could be wrong here, but it's possible we'll need to add another line here to explicitly export the plugins as well. Something like:
`export * as KeyManagerPlugins from "./walletSdk/KeyManager/plugins";`

Reply:
ah good catch, thx!
https://github.com/stellar/typescript-wallet-sdk/pull/98/commits/30b7247cedd1e40eca6634ffbbc8d9f20ab7081b